tests/test_models.py

```diff
@@ -711,26 +711,26 @@ def test_branding_configuration_saving_successfully(self):
             self.assertEqual(EnterpriseCustomerBrandingConfiguration.objects.count(), 2)
 
     @ddt.data(
-        (False, 2048 * 1024),
-        (False, 1024 * 1024),
-        (True, 512 * 1024),
-        (True, 256 * 1024),
-        (True, 128 * 1024),
+        (False, 2048),
+        (False, 1024),
+        (True, 512),
+        (True, 256),
+        (True, 128),
     )
     @ddt.unpack
-    def test_image_size(self, valid_image, image_size):
+    def test_image_size(self, is_valid_image_size, image_size):
         """
-        Test image size, image_size < (4 * 1024) e.g. 4kb. See apps.py.
+        Test image size in KB's, image_size < (4 * 1024) e.g. 4KB. See apps.py.
```

Comment:
The string here still refers to the old base limit. Please update that string to incorporate the new 512KB limit, as well as some language indicating that the 512KB limit is just the default.